src/lib/annotations/drawing/DrawingPath.js

```diff
@@ -0,0 +1,101 @@
+class DrawingPath {
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+    /** @property {Array} - The array of coordinates that form the path */
+    path;
+
+    /** @property {number} - The maximum X position of all coordinates */
+    maxX;
+
+    /** @property {number} - The maximum Y position of all coordinates */
+    maxY;
+
+    /** @property {number} - The minimum X position of all coordinates */
+    minX;
+
+    /** @property {number} - The minimum Y position of all coordinates */
+    minY;
+
+    /**
+     * [constructor]
+     * @return {DrawingPath} DrawingPath instance
+     */
+    constructor() {
+        this.path = [];
+        this.maxX = -Infinity;
```

Comment:
You can set all of these defaults in the declarations above :) Then you can nuke the constructor!